src/strings/en-us.json

```
@@ -28,8 +28,10 @@
     "AllowSubtitleManagement": "Allow this user to edit subtitles",
     "AllowFfmpegThrottling": "Throttle Transcodes",
     "AllowFfmpegThrottlingHelp": "When a transcode or remux gets far enough ahead from the current playback position, pause the process so it will consume fewer resources. This is most useful when watching without seeking often. Turn this off if you experience playback issues.",
+    "AllowFmp4TranscodingContainerHelp": "Allow fmp4 transcoding container for this tuner to enable HEVC and HDR contents. Not all tuners are compatible with this container. Disable this if you experience playback issues.",
```

We use `fMP4` for the "prefer" option. Shouldn't we do the same here?
```suggestion
    "AllowFmp4TranscodingContainerHelp": "Allow fMP4 transcoding container for this tuner to enable HEVC and HDR contents. Not all tuners are compatible with this container. Disable this if you experience playback issues.",
```